### PR TITLE
handling notification using global script

### PR DESCRIPTION
**Pull request by snehmehta**
closes: https://github.com/EnsembleUI/ensemble/issues/1289

Getting `navigateScreen` payload of `handleNotification` from the global script whose path is specified in environment variable.

### CODE REVIEW

File: lib/framework/notification_manager.dart [hunk 1]
```diff
@@ -120,37 +124,66 @@ class NotificationManager {
...
     });
   }
 
-  Future<void> handleNotification(RemoteMessage message) async {
-    Map<String, dynamic> payload = {
-      'notificationPayload': {
-        'title': message.notification?.title,
-        'body': message.notification?.body,
-        'data': message.data,
+  Future<void> handleNotification(String message) async {
```
Comment (vusters) [inline lib/framework/notification_manager.dart:134]: What about the existing customers with the existing format. They'll break

**Review by vusters — CHANGES_REQUESTED** [review]

**Review by vusters — APPROVED** [review]
This looks find, but please test Tin's use case to make sure it still work.

**Comment by snehmehta** [comment]
okay

**Comment by kmahmood74** [comment]
@snehmehta please merge and close if you have tested the Tin changes, we need to close PRs quickly after they have been approved

**Pull request merged**